test/uk/gov/hmrc/gatekeeper/services/ApplicationServiceSpec.scala

```diff
@@ -656,14 +580,14 @@ class ApplicationServiceSpec extends AsyncHmrcSpec with ResetMocksAfterEachTest
       CommandConnectorMock.IssueCommand.succeeds()
 
       val emailAddress = "email@example.com".toLaxEmail
-      val application  = stdApp1.copy(deployedTo = Environment.PRODUCTION)
+      val application  = standardApp.withEnvironment(Environment.PRODUCTION)
```

Comment:
Can we leave out `withEnvironment(Environment.PRODUCTION)` here and in 5 others places (4 in this file and one in SubscriptionFieldsServiceSpec), since the standardApp is already in production?

Maybe also inline values where it's only used a few times?

Reply:
We can